src/main/java/frc/team1816/robot/LogThread.java

```diff
@@ -0,0 +1,52 @@
+package frc.team1816.robot;
+
+import badlog.lib.BadLog;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Threads;
+import edu.wpi.first.wpilibj.Timer;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class LogThread extends Thread {
+    private BadLog logger;
+
+    public LogThread() {
+        super();
+        // Reduce the priority of this thread
+        Threads.setCurrentThreadPriority(true, 30);
+    }
+
+    public synchronized void initLog() {
+        // Format timestamp according to ISO 8601 e.g. 2019-02-14T16-37
+        var timestamp = new SimpleDateFormat("yyyy'-'MM'-'dd'T'HH'-'mm").format(new Date());
+
+        String defaultPath = "/home/lvuser/";
+        String usbPath = "/media/sda1/";
+        File f = new File(usbPath);
+        String path = (f.exists() && f.isDirectory() ? usbPath : defaultPath);
```

Comment:
We've been having some issues with writing to USB. I would comment it out until we have a chance to test more.

Also, I think it might work best to have the `path` String be `/home/lvuser/` by default, and just modify it if USB path is valid. In other words, refactor `defaultPath` as `path`

Reply:
Resolved in 6ee7e1cdd3ac4e87d46f5e55011dae3eeff132d6.